data/class/SC_FormParam.php

```diff
@@ -138,7 +138,7 @@ public function setHtmlDispNameArray()
             } else {
                 $this->html_disp_name[$index] = $this->disp_name[$index];
             }
-            if ($this->arrDefault[$key] != '') {
+            if (strlen($this->arrDefault[$key]) >= 1) {
```

Comment:
`$this->arrDefault` の添字に `$key` が存在しないと `E_NOTICE` が発生するので、以下でいかがでしょう？

```suggestion
            if (array_key_exists($key, $this->arrDefault) && strlen($this->arrDefault[$key]) >= 1) {
```

Reply:
E_NOTICE は黙殺で良いと認識しています。
変更前の方が可読性が高く、できれば変更したくないです。
(変更するにしても、同様のコード全てを変更して欲しいです。)

Reply:
全て変更するのは影響が甚大なので、変更なしにしましょう。